package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "lighthouse-stack-packs",
```

Comment:
Unsure what the best naming of this package should be.

Reply:
sgtm